src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -327,4 +351,178 @@ public static OpenLoadFlowParameters get(LoadFlowParameters parameters) {
+    }
+
+    static LfNetworkParameters getNetworkParameters(LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt,
+                                                    SlackBusSelector slackBusSelector, boolean breakers) {
+        return new LfNetworkParameters(slackBusSelector,
+                                       parametersExt.hasVoltageRemoteControl(),
+                                       parametersExt.getLowImpedanceBranchMode() == OpenLoadFlowParameters.LowImpedanceBranchMode.REPLACE_BY_MIN_IMPEDANCE_LINE,
+                                       parameters.isTwtSplitShuntAdmittance(),
+                                       breakers,
+                                       parametersExt.getPlausibleActivePowerLimit(),
+                                       parametersExt.isAddRatioToLinesWithDifferentNominalVoltageAtBothEnds(),
+                                       parameters.getConnectedComponentMode() == LoadFlowParameters.ConnectedComponentMode.MAIN,
+                                       parameters.getCountriesToBalance(),
+                                       parameters.isDistributedSlack() && parameters.getBalanceType() == LoadFlowParameters.BalanceType.PROPORTIONAL_TO_CONFORM_LOAD,
+                                       parameters.isPhaseShifterRegulationOn(),
+                                       parameters.isTransformerVoltageControlOn(),
+                                       parametersExt.isVoltagePerReactivePowerControl(),
+                                       parametersExt.hasReactivePowerRemoteControl(),
+                                       parameters.isDc(),
+                                       parameters.isSimulShunt(),
+                                       !parameters.isNoGeneratorReactiveLimits());
+    }
+
+    public static AcLoadFlowParameters createAcParameters(Network network, LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt,
+                                                          MatrixFactory matrixFactory, Reporter reporter) {
+        return createAcParameters(network, parameters, parametersExt, matrixFactory, reporter, false, false);
+    }
+
+    public static AcLoadFlowParameters createAcParameters(Network network, LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt,
+                                                          MatrixFactory matrixFactory, Reporter reporter, boolean breakers, boolean forceA1Var) {
+        AcLoadFlowParameters acParameters = createAcParameters(parameters, parametersExt, matrixFactory, reporter, breakers, forceA1Var);
+        if (parameters.isReadSlackBus()) {
+            acParameters.getNetworkParameters().setSlackBusSelector(new NetworkSlackBusSelector(network, acParameters.getNetworkParameters().getSlackBusSelector()));
+        }
+        return acParameters;
+    }
+
+    public static AcLoadFlowParameters createAcParameters(LoadFlowParameters parameters, OpenLoadFlowParameters parametersExt,
```

Comment:
Why is this class public? Finally only the two previous ones are used in the rest of the code.

Reply:
It is used in another private project